### PR TITLE
[PAYSHIP-1585] Added payment method disabling and warning messages for merchants who can't receive payments

### PR DESCRIPTION
**Pull request by L3RAZ**

### CODE REVIEW

File: translations/fr.php [hunk 1]
```diff
@@ -140,11 +139,11 @@
...
 $_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_cba316dfd19bdaf1c654cd84d82688fe'] = 'Cette transaction est refusée pour des raisons de violation de conformité.';
-$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_d00df53f04b51b8bba7060021a13250a'] = 'Ce client n\'est pas autorisé à utiliser  l\'objet &quot;payment_instriction&quot;.';
-$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_dddf233eaa8da4b85ebc33661ca4fe94'] = 'La devise de l\'autorisation de paiement doit être la même que la devise utilisée pour créer la commande et autorisée par le client. Vérifier le &quot;currency_code&quot; et essayer de nouveau.';
+$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_d00df53f04b51b8bba7060021a13250a'] = 'Ce client n\'est pas autorisé à utiliser  l\'objet &amp;amp;amp;quot;payment_instriction&amp;amp;amp;quot;.';
+$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_dddf233eaa8da4b85ebc33661ca4fe94'] = 'La devise de l\'autorisation de paiement doit être la même que la devise utilisée pour créer la commande et autorisée par le client. Vérifier le &amp;amp;amp;quot;currency_code&amp;amp;amp;quot; et essayer de nouveau.';
```
Comment (Matt75) [inline translations/fr.php:143]:
```suggestion
$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_dddf233eaa8da4b85ebc33661ca4fe94'] = 'La devise de l\'autorisation de paiement doit être la même que la devise utilisée pour créer la commande et autorisée par le client. Vérifier le currency_code et essayer de nouveau.';
```

File: translations/fr.php [hunk 2]
```diff
@@ -140,11 +139,11 @@
...
-$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_e39b765a3de4545e91f889374c84535c'] = 'L\'option pour enregistrer la commande est seulement disponible si le \'intent\' est AUTHORIZE et le &quot;processing_instruction&quot; est ORDER_SAVED_EXPLICITY. Veuillez changer le &quot;intent&quot; en AUTHORIZE, &quot;processing_instruction&quot; en &quot;ORDER_SAVED_EXPLICITY&quot; et essayer de nouveau.';
+$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_e39b765a3de4545e91f889374c84535c'] = 'L\'option pour enregistrer la commande est seulement disponible si le \'intent\' est AUTHORIZE et le &amp;amp;amp;quot;processing_instruction&amp;amp;amp;quot; est ORDER_SAVED_EXPLICITY. Veuillez changer le &amp;amp;amp;quot;intent&amp;amp;amp;quot; en AUTHORIZE, &amp;amp;amp;quot;processing_instruction&amp;amp;amp;quot; en &amp;amp;amp;quot;ORDER_SAVED_EXPLICITY&amp;amp;amp;quot; et essayer de nouveau.';
```
Comment (Matt75) [inline translations/fr.php:146]:
```suggestion
$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_e39b765a3de4545e91f889374c84535c'] = 'L\'option pour enregistrer la commande est seulement disponible si le \'intent\' est AUTHORIZE et le processing_instructionest ORDER_SAVED_EXPLICITY. Veuillez changer le intent en AUTHORIZE, processing_instruction en ORDER_SAVED_EXPLICITY et essayer de nouveau.';
```

File: translations/fr.php [hunk 3]
```diff
@@ -179,7 +178,7 @@
 $_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_a44217022190f5734b2f72ba1e4f8a79'] = 'Numéro de carte';
 $_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_bc9189406be84ec297464a514221406d'] = 'XXX';
 $_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_c884ed19483d45970c5bf23a681e2dd2'] = 'Voici un résumé de votre commande :';
-$_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur &quot;Je valide ma commande&quot;';
+$_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur &amp;amp;amp;quot;Je valide ma commande&amp;amp;amp;quot;';
```
Comment (Matt75) [inline translations/fr.php:181]:
```suggestion
$_MODULE['<{ps_checkout}prestashop>paymentcardconfirmation_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur Je valide ma commande';
```

File: translations/fr.php [hunk 4]
```diff
@@ -245,7 +244,7 @@
 $_MODULE['<{ps_checkout}prestashop>ps_checkout_c884ed19483d45970c5bf23a681e2dd2'] = 'Voici un résumé de votre commande :';
 $_MODULE['<{ps_checkout}prestashop>ps_checkout_dbb50a3167fd387cac23808bd55a9ee4'] = 'Référence de Transaction PayPal';
 $_MODULE['<{ps_checkout}prestashop>ps_checkout_e81c4e4f2b7b93b481e13a8553c2ae1b'] = 'ou';
-$_MODULE['<{ps_checkout}prestashop>ps_checkout_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur &quot;Je valide ma commande&quot;';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur &amp;amp;amp;quot;Je valide ma commande&amp;amp;amp;quot;';
```
Comment (Matt75) [inline translations/fr.php:247]:
```suggestion
$_MODULE['<{ps_checkout}prestashop>ps_checkout_edd87c9059d88fea45c0bd6384ce92b9'] = 'Veuillez finaliser votre commande en cliquant sur Je valide ma commande';
```

**Review by Matt75 — COMMENTED** [review]
Please review and remove encoded chars

File: translations/de.php [hunk 5]
```diff
@@ -308,7 +307,7 @@
...
+$_MODULE['<{ps_checkout}prestashop>translations_26ec71edbfce31656c47a66bf830ed1c'] = 'Achtung, Ihre gerundeten Parameter sind nicht völlig kompatibel mit der Transaktionsverarbeitung von PrestaShop Checkout. Einige Transaktionen können möglicherweise fehlschlagen. Aber es ist ganz einfach, Ihre Einstellungen für &amp;amp;amp;quot;Rundungsregel&amp;amp;amp;quot; und &amp;amp;amp;quot;Rundungstyp&amp;amp;amp;quot; müssen auf &amp;amp;amp;quot;Bei halbem Wert auf unendlich runden&amp;amp;amp;quot; und &amp;amp;amp;quot;Für jedes Element runden&amp;amp;amp;quot; eingestellt werden, oder klicken Sie auf die Schaltfläche unten, um dies automatisch zu tun!';
```
[the remark this answers is not in the document]
Comment (seiwan) [reply translations/de.php]: Done

File: translations/de.php [hunk 6]
```diff
@@ -411,7 +410,7 @@
...
 $_MODULE['<{ps_checkout}prestashop>translations_88bccdecc2c4b9dcc8afbbc94483293c'] = 'von dem Know-how und den Vorteilen von PayPal profitieren';
 $_MODULE['<{ps_checkout}prestashop>translations_8a9f98e889a0de9caf2feb4b9b382284'] = 'Mit Ihrer Abmeldung deaktivieren Sie alle Zahlungsmethoden. Sie können jetzt keine Bezahlungen mehr über Ihr PrestaShop-Checkout-Konto erhalten.';
-$_MODULE['<{ps_checkout}prestashop>translations_8acc0bc61a5f75b13cea7e27bbcd51af'] = 'Sie können die Art der Kreditkartenfelder nur dann auswählen, wenn die Option Kreditkarte auf der Registerkarte &quot;Checkout-Erfahrung anpassen&quot; aktiviert ist.';
+$_MODULE['<{ps_checkout}prestashop>translations_8acc0bc61a5f75b13cea7e27bbcd51af'] = 'Sie können die Art der Kreditkartenfelder nur dann auswählen, wenn die Option Kreditkarte auf der Registerkarte &amp;amp;amp;quot;Checkout-Erfahrung anpassen&amp;amp;amp;quot; aktiviert ist.';
```
[the remark this answers is not in the document]
Comment (seiwan) [reply translations/de.php]: Done

File: translations/fr.php (hunk 4 above)
Comment (seiwan) [reply translations/fr.php]: Done

File: translations/fr.php (hunk 3 above)
Comment (seiwan) [reply translations/fr.php]: Done

File: translations/fr.php (hunk 2 above)
Comment (seiwan) [reply translations/fr.php]: Done

File: translations/fr.php (hunk 1 above)
Comment (seiwan) [reply translations/fr.php]: Done

File: translations/fr.php [hunk 7]
```diff
@@ -140,11 +139,11 @@
...
 $_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_c4cdc3c3b50c0a7aa017266797228972'] = 'Le traitement de ce type de carte n\'est pas pris en charge. Veuillez utiliser un autre type de carte.';
 $_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_cba316dfd19bdaf1c654cd84d82688fe'] = 'Cette transaction est refusée pour des raisons de violation de conformité.';
-$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_d00df53f04b51b8bba7060021a13250a'] = 'Ce client n\'est pas autorisé à utiliser  l\'objet &quot;payment_instriction&quot;.';
-$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_dddf233eaa8da4b85ebc33661ca4fe94'] = 'La devise de l\'autorisation de paiement doit être la même que la devise utilisée pour créer la commande et autorisée par le client. Vérifier le &quot;currency_code&quot; et essayer de nouveau.';
+$_MODULE['<{ps_checkout}prestashop>hostedfieldserrors_d00df53f04b51b8bba7060021a13250a'] = 'Ce client n\'est pas autorisé à utiliser  l\'objet &amp;amp;amp;quot;payment_instriction&amp;amp;amp;quot;.';
```
[the remark this answers is not in the document]
Comment (seiwan) [reply translations/fr.php]: Done

**Review by Matt75 — APPROVED** [review]

**Pull request merged**